medusa/search/core.py

```diff
@@ -473,6 +473,8 @@ def search_for_needed_episodes(force=False):
             # fail_over_delay time (hours) skipp it. For this we need to check if the result has already been
             # stored in the provider cache db, and if it's still younger then the providers attribute fail_over_delay.
             if cur_provider.fail_over_enabled and cur_provider.fail_over_hours:
+                log.debug('DELAY: Provider {provider} delay enabled, with an expiration of {delay}',
+                          {'provider': cur_provider.name, 'delay': cur_provider.fail_over_hours})
```

Comment:
maybe add `hours` here? or the correct unit?

Reply:
ooh yeah sure.